builtin/logical/pki/cert_util.go

```diff
@@ -1577,27 +1576,32 @@ func generateCreationBundle(b *backend, data *inputBundle, caSign *certutil.CAIn
 }
 
 // compute a certificate's Not Before based on the role and input api data sent. Returns notBefore Time and an error.
-func getCertificateNotBefore(data *inputBundle) (time.Time, error) {
+func getCertificateNotBefore(data *inputBundle) (time.Time, []string, error) {
```

Comment:
If we are not going to use the warnings now I would rather not have this function return it, easier to initialize a slice on the caller.